### PR TITLE
Fix issues #6268

### PR DESCRIPTION
**Pull request by davi19**
- Change file ContentReader, SongReader,VideoReader

### CODE REVIEW

File: IDE/VisualStudioForMac/Properties/AddinInfo.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 using System;
 using Mono.Addins;
 using Mono.Addins.Description;
-[assembly: Addin("MonoGame_IDE_VisualStudioForMac",Namespace = "MonoDevelop",Version = "0.0.0.0")]
+[assembly: Addin("MonoGame_IDE_VisualStudioForMac",Namespace = "MonoDevelop",Version = "3.7.0.0")]
```
Comment (cra0zy) [inline IDE/VisualStudioForMac/Properties/AddinInfo.cs]: This number is getting set by the build system and should not be modified.

File: MonoGame.Framework/Content/ContentReader.cs [hunk 2]
```diff
@@ -118,7 +118,7 @@ public T ReadExternalReference<T>()
 
             if (!String.IsNullOrEmpty(externalReference))
             {
-                return contentManager.Load<T>(FileHelpers.ResolveRelativePath(assetName, externalReference));
+                return contentManager.Load<T>(MonoGame.Utilities.FileHelpers.ResolveRelativePath(assetName, externalReference));
```
Comment (cra0zy) [inline MonoGame.Framework/Content/ContentReader.cs]: Use a `using Namespace`

File: IDE/VisualStudioForMac/Properties/AddinInfo.cs (hunk 1 above)
Comment (davi19) [reply IDE/VisualStudioForMac/Properties/AddinInfo.cs]: ok, the visual studio code change automaticly when i opened the project. I'm gonna ignore next time

File: MonoGame.Framework/Content/ContentReader.cs (hunk 2 above)
Comment (davi19) [reply MonoGame.Framework/Content/ContentReader.cs]: Ok. thanks i'm go refer the name space. Thanks for reply.

**Comment by Jjagg** [comment]
Superseded by #6488.

**Pull request closed without merging**